README.md

```diff
@@ -1,4 +1,4 @@
-# Aphrodite: Inline Styles that work
+# Aphrodite: CSS in JS, but with real CSS
```

Comment:
The description on GitHub has since been updated to "Framework-agnostic CSS-in-JS with support for server-side rendering, browser prefixing, and minimum CSS generation" (#259). @kentcdodds if you update your PR to match I'd be happy to merge it in!

Reply:
Done!